test/elements/CardElement.test.tsx

```diff
@@ -70,7 +76,8 @@ describe('CardElement', () => {
         disabled,
         autoComplete,
       },
-      undefined
+      undefined,
+      ref
     );
     expect(useListener).toHaveBeenCalledWith('ready', element, onReady);
     expect(useListener).toHaveBeenCalledWith('change', element, onChange);
```

Comment:
Maybe check the current value of the passed ref?
Same for tests below

Reply:
Maybe test that it doesn't explode when a ref is not passed.

Reply:
I can't check the `ref` value w/ jest, its always `null` no matter what, couldn't find anything online/in the docs about it.
Doesn't seem like the render method in the testing library forwards the `ref` properly.
I tested w/ real usage and unit on the useElement hook, but 
From what I saw online people have been using [enzyme](https://enzymejs.github.io/enzyme/) for that

Reply:
For testing w/ no ref, should I do a new test on each element? What do you suggest?
It doesn't explode, I tested it both ways, but what do you suggest on formalizing it?
I'm just thinking about not having a lot of repeated code...

Reply:
You `useElement` tests cover for the first case I think. 
For the second, you could add chance to initialize or not the ref, since that doesn't change the snapshot.